src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -278,6 +292,9 @@ private static String getTwoWindingsTransformerFieldValue(FieldType field, TwoWi
             case PHASE_REGULATING,
                 PHASE_REGULATION_MODE,
                 PHASE_REGULATION_VALUE -> getPhaseTapChangerFieldValue(field, twoWindingsTransformer.getPhaseTapChanger());
+            case FREE_PROPERTIES -> twoWindingsTransformer.getProperty(propertyName);
```

Comment:
not needed as we add a case in getFieldValue

Reply:
done